<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>300</height></rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Times New Roman; font-size: 13px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>60</height></size></property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <item>
        <widget class="QLabel" name="lb_title">
         <property name="text"><string>Вход в систему</string></property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_login">
         <property name="placeholderText"><string>Логин</string></property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_password">
         <property name="placeholderText"><string>Пароль</string></property>
         <property name="echoMode"><enum>QLineEdit::Password</enum></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_login">
         <property name="text"><string>Войти</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_guest">
         <property name="text"><string>Войти как гость</string></property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_bot">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>60</height></size></property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
